libraries/scripting/common/src/kotlin/script/experimental/api/scriptData.kt

```diff
@@ -53,6 +55,21 @@ interface SourceCode {
     data class Location(val start: Position, val end: Position? = null) : Serializable
 }
 
+/**
+ * Annotation found during script source parsing along with its location
+ */
+data class CollectedScriptAnnotation<out A : Annotation>(
```

Comment:
I do not like the name somehow. I feel to me that something like `ScriptSourceAnnotations` or `SourceCodeAnnotations` will sound better.

Reply:
I like `ScriptSourceAnnotation`

Reply:
✅